lib/schedule/minischedule/piece.ex

```diff
@@ -14,6 +14,8 @@ defmodule Schedule.Minischedule.Piece do
           mid_route?: boolean()
         }
 
+  @type timepoint_names_by_id :: %{String.t() => String.t()}
```

Comment:
This is also defined identically in `trip.ex` and in `data.ex` as
```ex
   @type timepoint_names_by_id :: %{Timepoint.id() => String.t() | nil} 
```

Why is the null different?

If this definition was moved to `timepoint.ex`, then all the files could share the definition.

Reply:
Actually, looking at this, the `nil` is wrong and the version without the `nil` should be used. This is a fine example of the dangers of repeating yourself. Will correct.